### PR TITLE
Marker component added

### PR DESCRIPTION
**Pull request by swa07016**
- Marker component added.
- KakaoMap zoomable feature added.

### CODE REVIEW

File: src/KakaoMap.js [hunk 1]
```diff
@@ -85,53 +43,33 @@ class KakaoMap extends React.Component {
...
             });
         };
     };
 
     render() {
-        return <Root id="map" />;
+        const { map } = this.state;
+        return (
+            <Root id="map">
+                {map ? (
```
Comment (soaple) [inline src/KakaoMap.js]: How about change this code to use && operator?

For example,
`{map && <Component />}`

File: src/Marker.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                    ? this.createMarkerImg(
+                            markerImgSrc,
+                            markerImgWidth,
+                            markerImgHeight
+                        )
+                    : '',
+        });
+        details
+            ? this.createCustomOverlay(this.context, marker, details)
+            : '';
```
Comment (soaple) [inline src/Marker.js]: How about change this code to use 'if' statement?

For example,
```
if (details) {
    this.createCustomOverlay(this.context, marker, details)
}
```

File: src/Marker.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            : ''; 
+        marker.setMap(this.context);
+    }
+
+    createMouseClickListener = (map, customOverlay) => {
+        let overlay = false;
+        const toggle = () => (overlay = !overlay);
+        return () => {
+            toggle();
+            overlay ? customOverlay.setMap(map) : customOverlay.setMap(null);
```
Comment (soaple) [inline src/Marker.js]: You can change this code to simple like below.

`customOverlay.setMap(overlay ? map : null);`

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the comments.

**Comment by swa07016** [comment]
I fixed some codes.

**Review by soaple — APPROVED** [review]
LGTM

**Pull request merged**